ci/script.sh

```diff
@@ -6,7 +6,7 @@ cargo build --target ${TARGET}
 cargo test --target ${TARGET}
 
 # On Rust 1.31.0, we only care about passing tests.
-if [ ! rustc --version | grep "^rustc 1.31.0" ]; then
+if [[ ! $(rustc --version | grep "^rustc 1.31.0") ]]; then
```

Comment:
Change this to:

```
if [ "$TRAVIS_RUST_VERSION" = "stable" ]; then
```

Btw: `[[` is a bash-ism. Of course, we have a bash she-bang, but it does makes scripts less portable.